src/helpers/get-week-number.ts

```diff
@@ -1,9 +1,15 @@
-export const getWeekNumber = (d: Date) => {
+export const getYearWeek = (d: Date) => {
   d = new Date(Date.UTC(d.getFullYear(), d.getMonth(), d.getDate()));
+
   d.setUTCDate(d.getUTCDate() + 4 - (d.getUTCDay() || 7));
   const yearStart = new Date(Date.UTC(d.getUTCFullYear(), 0, 1));
+
   const weekNo = Math.ceil(
     ((d.getTime() - yearStart.getTime()) / 86400000 + 1) / 7
   );
-  return [d.getUTCFullYear(), weekNo];
+
+  return [
+    d.getUTCFullYear(),
+    weekNo.toString(10).length === 1 ? `0${weekNo}` : weekNo,
```

Comment:
Maybe we can make use of [`padStart()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart)?

```suggestion
    weekNo.toString(10).padStart(2, '0'),
```